ClosedXML/Excel/Ranges/XLRangeBase.cs

```diff
@@ -513,18 +513,21 @@ public virtual Boolean IsEmpty(Boolean includeFormats)
 
         public virtual Boolean IsEntireRow()
         {
-            return RangeAddress.FirstAddress.ColumnNumber == 1
-                   && RangeAddress.LastAddress.ColumnNumber == XLHelper.MaxColumnNumber;
+            return RangeAddress.IsEntireRow();
         }
 
         public virtual Boolean IsEntireColumn()
         {
-            return RangeAddress.FirstAddress.RowNumber == 1
-                   && RangeAddress.LastAddress.RowNumber == XLHelper.MaxRowNumber;
+            return RangeAddress.IsEntireColumn();
+        }
+
+        public virtual Boolean IsEntireSheet()
+        {
+            return RangeAddress.IsEntireSheet();
```

Comment:
You made these methods `virtual` to override them in XLRow, XLColumn, XLWorksheet, I assume. Did you forget this?

Reply:
Ah, yes, the `IsEntireSheet()` method doesn't need to be virtual. Copy and paste error.